CONTRIBUTING.md

```diff
@@ -116,11 +116,12 @@ In order to create a branch, either fork the repository or request access to the
 Be sure to link the issue you created earlier in the pull request.
 - In the bottom, you can select to either submit as a pull request or as a draft pull request.
 - Use a draft pull request first, so the community can already see the ongoing work.
+- In this stage, the review from parson takes place.
   
 Also, the CI pipeline will run for every commit, so you can continuously check your work.
 
 ### Convert into 'real' pull request
 
-Once you are done with your changes, [convert the draft to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
+Once you are done with your changes and the review from parson took place, [convert the draft to a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request).
```

Comment:
see above

Reply:
Agreed. Changed it to "service provider".